app/controllers/concerns/account_controller_concern.rb

```diff
@@ -41,6 +42,13 @@ def atom_account_url_link
     ]
   end
 
+  def actor_url_link
+    [
+      ActivityPub::TagManager.instance.uri_for(@account),
+      [%w(rel alternate), %w(type application/activity+json)],
```

Comment:
should be `application/ld+json; profile="https://www.w3.org/ns/activitystreams"` throughout.